### PR TITLE
Add example of argument object pattern.

### PR DESCRIPTION
**Pull request by harperj**
People who might have an opinion, offhand: @slloyd @panthomakos @bdejesus @marexandre

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,40 @@ $container = $('#container')
...
+
+class Foo
+  constructor: (args) ->
+    {
+      @varA,
+      @varB,
+      @varC,
+      @varD,
+      @varE,
+      @varF
```
Comment (stravabr) [inline README.md]: Maybe include an example of a default value for one of these (`@varF = 'foo'`)?

Comment (harperj) [reply README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -233,22 +233,21 @@ class Foo
...
+)
 
 
 ### Bad. ###
 class Bar
-  constructor: (@varA, @varB, @varC, @varD, @varE, @varF) ->
+  constructor: (@varA, @varB, @varC, @varD, @varE, @varF = 'defaultString') ->
     ...
 
 bar = new Bar(inputA, inputB, inputC, inputD, inputE, inputF)
```
Comment (stravabr) [inline README.md]: You could omit `inputF` from here now too.

Comment (harperj) [reply README.md]: Done.

**Review by stravabr — APPROVED** [review]
I'm in favor.

**Pull request closed without merging**